pkgs/applications/version-management/sourcehut/default.nix

```diff
@@ -64,6 +65,13 @@ let
           inherit version;
           hash = "sha256-VUslfHS763oNJUFgpPj/4YUkP1KlIDUGC3Ycpi2XfwM=";
         };
+        # Fixes a test failure with Pytest 8
+        patches = (oldAttrs.patches or []) ++ [
+          (fetchpatch {
+            url = "https://github.com/pallets/werkzeug/commit/4e5bdca7f8227d10cae828f8064fb98190ace4aa.patch";
+            hash = "sha256-H45/YF9zaOUg6UqEEus4uBeGA/TjynuJZcRyc6BHQ30=";
```

Comment:
this hash seems wrong and should be:
```
 hash = "sha256-83doVvfdpymlAB0EbfrHmuoKE5B2LJbFq+AY2xGpnl4=";
```
see: https://hydra.nixos.org/build/259069084
and
https://malob.github.io/nix-review-tools-reports/nixpkgs:trunk/nixpkgs_trunk_1806175.html

verified that updating the hash allows `nix-build -A sourcehut.todosrht` to succeed.

Reply:
Confirmed this, will send in a PR to fix it.

Reply:
PR submitted: #310250